.gitignore

```diff
@@ -18,6 +18,7 @@ migrate_working_dir/
 /android/.idea/*
 !/.idea/inspectionProfiles/
 !/android/.idea/inspectionProfiles/
+.fvm
```

Comment:
This seems potentially useful. It should go in its own commit, though, with its own commit message explaining the motivation for it. See the Zulip project style guide on clean commit history:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html